docs/fundamentals/networking/http/httpclient-guidelines.md

```diff
@@ -52,37 +52,41 @@ To summarize recommended `HttpClient` use in terms of lifetime management, you s
 
 For more information about managing `HttpClient` lifetime with `IHttpClientFactory`, see [`IHttpClientFactory` guidelines](../../../core/extensions/httpclient-factory.md#httpclient-lifetime-management).
 
-## Resilience policies with static clients
+## Resilience with static clients
```

Comment:
We _might_ want to keep the old titles in case there were links to the section... "Policies" sound like an ordinary word, so IMHO it looks ok despite the method names not containing "Policy" anymore... cc @IEvangelist @gewarren for opinion

Reply:
The `Policies` were a term used in Polly v8, that's no longer relevant in V8. That's why I just dropped it from the section name.

Reply:
If there were links to the section, they'd just go to the top of the page now since the bookmark isn't found, so it's not the end of the world to change it.

Reply:
I agree that it should be dropped as they're no longer referred to as policies, that is legacy terminology as @martintmk alluded to.